apps/controllo-ore-x/src/app/_shared/components/sidenav/sidenav.component.ts

```diff
@@ -1,22 +1,41 @@
-import { Component, OnInit } from '@angular/core';
-import { NavMenusVisibilityService } from './servicies/nav-menus-visibility.service';
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { ADMINISTRATION_MENU_SECTIONS } from '@app/_core/config/administration-menu-sections.config';
+import { MAIN_MENU_SECTIONS } from '@app/_core/config/main-menu-sections.config';
+import { MANAGEMENT_MENU_SECTIONS } from '@app/_core/config/management-menu-sections.config';
+import { IMenuSections } from '@app/_core/interfaces/i-menu-sections.interface';
+import {
+  SubscriptionsLifecycle,
+  completeSubscriptions,
+} from '@app/utils/subscriptions_lifecycle';
 import { Subscription } from 'rxjs';
-import { SubscriptionsLifecycle, completeSubscriptions } from '@app/utils/subscriptions_lifecycle';
+import { NavMenusVisibilityService } from './servicies/nav-menus-visibility.service';
 
 @Component({
   selector: 'controllo-ore-x-sidenav',
   templateUrl: './sidenav.component.html',
   styleUrls: ['./sidenav.component.scss'],
 })
-export class SidenavComponent implements OnInit, SubscriptionsLifecycle {
+export class SidenavComponent implements OnInit, OnDestroy, SubscriptionsLifecycle {
+  MAIN_MENU_SECTIONS: IMenuSections[] = MAIN_MENU_SECTIONS;
+  ADMINISTRATION_MENU_SECTIONS: IMenuSections[] = ADMINISTRATION_MENU_SECTIONS;
+  MANAGEMENT_MENU_SECTIONS: IMenuSections[] = MANAGEMENT_MENU_SECTIONS;
+
   isSidenavOpen: boolean = true;
 
+  selectedSection: string;
```

Comment:
```suggestion
  activeSection: string;
```
Change also CSS classes to `active`; it better represents the UX and HTML (the elements are `li`, not radio buttons or other inputs that can be selected and deselected.